### ISSUE TITLE
Fix section 508 accessibility notice problems

### ISSUE THREAD

**Issue by msecret** [18F/openFEC#585]
Notice problems are less severe. They should be fixed but I believe the site is still compliant.

WCAG2AA.Principle1.Guideline1_3.1_3_1.H49.I (3 occurences)
   Semantic markup should be used to mark emphasised or special text so that it can be programmatically determined.
```
<i class="ti ti-help-alt"></i>
```

WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.BgImage (4 occurences)
   This element's text is placed on a background image. Ensure the contrast ratio between the text and all covered parts of the image are at least 3:1.

WCAG2AA.Principle1.Guideline1_3.1_3_1.H44.NotFormControl
   This label's "for" attribute contains an ID for an element that is not a form control. Ensure that you have entered the correct ID for the intended element.
```
<label for="search" class="text--hero">To get started, search for a fe...</label>
```

**Comment by noahmanger** [comment]
I believe this is all done, right? Re-open if not.
